src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	static final String Separator = ",|:";
+	static final String pattern = "//(.)\n(.*)";
+	static final String IntegerPattern = "^[0-9]*$";
+
+	public static int splitAndSum(String str) {
+		if (isBlankOrNull(str)) {
+			return 0;
+		}
+		if (isInteger(str)) {
+			return Integer.parseInt(str);
+		}
+		return getSum(split(str));
+	}
+
+	private static int getSum(String[] numbers) {
+		int result = 0;
+		int strToNumber;
+		for (String str : numbers) {
+			strToNumber = Integer.parseInt(str);
+			result += strToNumber;
+			if (!isInteger(str)) {
+				throw new RuntimeException();
+			}
+		}
+		return result;
```

Comment:
```suggestion

		for (String str : numbers) {
			int strToNumber = Integer.parseInt(str);
			result += strToNumber;
			if (!isInteger(str)) {
				throw new RuntimeException();
			}
		}
		return result;
```
for문 안으로 들어가도 괜찮을 것 같아요 :) 
str 같이 줄이지 않으면 좋을 것 같아요. 이름은 길어져도 괜찮습니다.